styles/Header.module.css

```diff
@@ -1,8 +1,333 @@
+    align-self: center;
+    padding-left: 30px;
+    flex: 1;
+}
+
+.searchForm {
+    display: flex;
+    width: 30%;
+    min-width: 18%!important;
+}
+
+.searchInput {
+    height: 30px;
+    margin: auto;
+    width: 80%;
+}
+
+.formButton{
+    align-self: center;
+    flex: 1 1 auto;
+    height: 30px;
+    cursor: pointer;
+    background-image: url("/magnifying-glass.svg");
+    background-position: center;
+    background-repeat: no-repeat;   
+    border-style: solid;
+    border-width: 1px;
+    background-size: 20%;
+}
+
+.menuIcon {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+    margin: 0 50px;
+    cursor: pointer;
+    display: none;
+    background-image: url("/menu.svg");
+    background-color: white;
+    background-size: 70%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-style: none; 
+  }
+
+.searchIcon {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+    margin: 0 50px;
+    font-size: 18px;
+    color:black;
+    cursor: pointer;
+    display: none;
+    background-image: url("/magnifying-glass.svg");
+    background-color: white;
+    background-size: 70%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-style: none; 
+  }
+
+
+.xIcon {
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  margin-top: 0px;
+  padding-left: 40px;
+  background-image: url("/cancel.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  border-style: none;
+  background-color: white;
+  background-size: 50%;
+  cursor: pointer;
+  min-width: 50px;
+  min-height: 50px;
+}
+
+.menuIcon span,
+.cancelIcon,
+.searchIcon {
+    display: none;
+}
+
+.mobileNavMenu {
+  position: fixed;
+  top: 0;
+  width: 200px;
+  height: 100vh;
+  padding: 30px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  left: -250px;
+  transition: all 0.3s;
+  box-shadow: 0px 5px 15px 0 rgba(0, 0, 0, 0.15);
+  display: none;
+}
+
+.mobileNavMenu.open {
+  left: 0;
+}
+
+.linksNav {
+  display: inline-block;
+  padding: 25px 15px;
+  text-decoration: none;
+  color: black;
+}
+
+.activeLinkNav {
+  color: #5CA2D5;
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.mobileNavSearch {
+  position: fixed;
+  top: 0;
+  width: 350px;
+  height: 50px;
+  padding: 30px;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  right: -350px;
+  top: 75px;
+  transition: all 0.3s;
+  box-shadow: 0px 15px 15px 0 rgba(0, 0, 0, 0.15);
+  display: none;
+}
+
+.mobileNavSearch.open {
+  right: 0;
+}
+
+.searchFormNav {
+  display: inline-block;
+  height: 30px;
+  width: 100%;
+}
+
+.formButtonNav {
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+  background-image: url("/magnifying-glass.svg");
+  background-position: center;
+  background-repeat: no-repeat;   
+  border-style: solid;
+  border-width: 1px;
+  background-size: 65%;
+}
+
+
+@media (max-width: 1245px) {
```

Comment:
This media query is unnecessary. It's not doing anything right now since the default style of navBar is applying the same `padding: 0 50px;` style. Remove please 🙂